client/ayon_core/plugins/publish/integrate.py

```diff
@@ -765,7 +765,7 @@ def prepare_representation(
             # Manage anatomy template data
             template_data.pop("frame", None)
             if is_udim:
-                template_data["udim"] = repre["udim"][0]
+                template_data["udim"] = repre["udim"]
```

Comment:
This is wrong and will break the templating for single UDIM files. Please revert.